File: aifupan_project/ReviewAnalysis/vo/video/ToGeneratedVo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ReviewAnalysis.Ai.Model;
using ReviewAnalysis.Dto;

namespace ReviewAnalysis.vo.video
{
    public class ToGeneratedVo
    {
        /// <summary>
        /// 来源id
        /// </summary>
        public string sourceId {  get; set; }

        /// <summary>
        /// 来源类型0视频、1文件、2对比分析
        /// </summary>
        public int sourceType { get; set; }
        /// <summary>
        /// 类型，1是自然原文，2是优化原文
        /// </summary>
        public int type { get; set; }
        /// <summary>
        /// 优化原文生成状态 0待生成，1生成中，2生成成功，3生成失败
        /// </summary>
        public int status { get; set; }

        /// <summary>
        /// 使用的aiModel
        /// </summary>
        public int aiModel { get; set; }

        /// <summary>
        /// 点击生成时的时间戳
        /// </summary>
        public long startDateTime { get; set; }
        /// <summary>
        /// 视频信息
        /// </summary>
        public VideoInfoVo anchorVideo { get; set; }
        /// <summary>
        /// 文件信息
        /// </summary>
        public UploadFileDto uploadFile { get; set; }
        /// <summary>
        /// 视频或文件内容
        /// </summary>
        public List<VideoContentVo> videoContentList { get; set; }

        /// <summary>
        /// 重写Equals方法，对比是否是同一个对象
        /// </summary>
        /// <param name="obj"></param>
        /// <returns></returns>
        public override bool Equals(object obj)
        {
            if (obj == null || GetType() != obj.GetType()) return false;
            return sourceId == ((ToGeneratedVo)obj).sourceId && type == ((ToGeneratedVo) obj).type;
        }
    }
}

File: aifupan_project/fupan-server/replay-words/src/main/java/com/jiuyu/replay/words/vo/video/ToGeneratedVo.java
package com.jiuyu.replay.words.vo.video;

import com.jiuyu.replay.generic.vo.words.AnchorVideoInfoVo;
import com.jiuyu.replay.generic.vo.words.VideoContentVo;
import com.jiuyu.replay.words.vo.file.UploadFileInfoVo;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.io.Serializable;
import java.util.List;

/**
 * @author ：lujie
 * @description：
 * @date ：2025/6/24 下午5:27
 */
@Schema(description = "获取当前用户待生产的自然、优化原文出参")
@Data
public class ToGeneratedVo implements Serializable {
    private static final long serialVersionUID = 1L;

    @Schema(description = "来源id")
    private String sourceId;

    @Schema(description = "类型 0视频，1文件")
    private Integer sourceType;

    @Schema(description = "原文类型 0分钟，1自然原文，2优化原文")
    private Integer type;

    @Schema(description = "使用的模型")
    private Integer aiModel;

    @Schema(description = "状态 0待生成，1生成中，2生成成功，3生成失败")
    private Integer status;

    @Schema(description = "点击生成时的时间戳")
    private Long startDateTime;

    @Schema(description = "视频信息")
    private AnchorVideoInfoVo anchorVideo;

    @Schema(description = "文件信息")
    private UploadFileInfoVo uploadFile;

    @Schema(description = "提示词")
    private List<VideoContentVo> videoContentList;

}
